Tasks/AzureVmssDeployment/make.json

```diff
@@ -7,6 +7,12 @@
             "compile" : true
         },
         {
+            "module": "../Common/azure-blobstorage-artifactProvider",
```

Comment:
Add rm -rf for node_modules/vsts-task-lib under this also

Reply:
done